plugins/oracledb_metrics.yaml

```diff
@@ -0,0 +1,649 @@
+              metric_name: oracle.tablespace.size
+              monotonic: "false"
+              static_attributes:
+                db.system: oracle
+                state: free
+              unit: by
+              value_column: FREE_SPACE
+              value_type: int
+            - attribute_columns:
+                - DATABASE_ID
+                - GLOBAL_NAME
+                - TABLESPACE_NAME
+                - CONTENTS
+              data_type: sum
+              description: The size of tablespaces in the database.
+              metric_name: oracle.tablespace.size
+              monotonic: "false"
+              static_attributes:
+                db.system: oracle
+                state: used
+              unit: by
+              value_column: USED_SPACE
+              value_type: int
+          sql: "SELECT (SELECT DBID FROM SYS.GV_$$DATABASE) DATABASE_ID, (SELECT GLOBAL_NAME FROM sys.GLOBAL_NAME) GLOBAL_NAME, ts.TABLESPACE_NAME, ts.CONTENTS,\n\t\t\t\t(select sum(df.bytes) from sys.dba_data_files df where df.tablespace_name=ts.tablespace_name)-(select sum(fs.bytes) from sys.dba_free_space fs where fs.tablespace_name=ts.tablespace_name) AS USED_SPACE,\n\t\t\t\t(select sum(fs.bytes) from sys.dba_free_space fs where fs.tablespace_name=ts.tablespace_name) AS FREE_SPACE\n\t\t\tFROM sys.dba_tablespaces ts \n\t\t\tWHERE ts.contents <> 'TEMPORARY'\n\t\t\tUNION ALL\n\t\t\tSELECT (SELECT DBID FROM SYS.GV_$$DATABASE) DATABASE_ID, (SELECT GLOBAL_NAME FROM sys.GLOBAL_NAME) GLOBAL_NAME, ts.NAME TABLESPACE_NAME, 'TEMPORARY' as CONTENTS,\n\t\t\t\t\tSUM(ss.USED_BLOCKS * t.BLOCK_SIZE) USED_SPACE, \n\t\t\t\t\tSUM(t.BYTES) - SUM(ss.USED_BLOCKS * t.BLOCK_SIZE) FREE_SPACE\n\t\t\tFROM SYS.V_$$sort_segment ss\n\t\t\tJOIN sys.v_$$tablespace ts\n\t\t\tON ss.TABLESPACE_NAME = ts.NAME\n\t\t\tJOIN sys.v_$$tempfile t\n\t\t\tON t.TS# = ss.TS#\n\t\t\tGROUP BY ts.NAME"
+        - metrics:
+            - attribute_columns:
+                - DATABASE_ID
+                - GLOBAL_NAME
+                - STATUS
+                - CONTENTS
+              data_type: sum
+              description: The number of tablespaces in the database.
+              metric_name: oracle.tablespace.count
+              monotonic: "false"
+              static_attributes:
+                db.system: oracle
+              unit: "{tablespaces}"
+              value_column: COUNT
+              value_type: int
+          sql: SELECT (SELECT DBID FROM SYS.GV_$$DATABASE) DATABASE_ID, (SELECT GLOBAL_NAME FROM sys.GLOBAL_NAME) GLOBAL_NAME, CONTENTS, STATUS, COUNT(*) COUNT FROM sys.dba_tablespaces GROUP BY STATUS, CONTENTS
+        - metrics:
+            - attribute_columns:
+                - DATABASE_ID
+                - GLOBAL_NAME
+              data_type: sum
+              description: The number of seconds since the last RMAN backup.
+              metric_name: oracle.backup.latest
+              monotonic: "true"
```

Comment:
Does this map to the bool true? I know that OTel does surprising things sometimes when they map to the config structs, so I'm just curious if this string is actually mapped to the bool value or not.

Reply:
It's a bool in the sqlqueryreceiver config [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6dc71bd739b4eaa407a9a596dc57ca4884e6313e/receiver/sqlqueryreceiver/config.go#L77). I'm guessing map structure attempts a string to bool conversion.